rta/shellcode_winexec_calc.py

```diff
@@ -0,0 +1,93 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0; you may not use this file except in compliance with the Elastic License
+# 2.0.
+
+from . import common
+from . import RtaMetadata
+from ctypes import *
+import ctypes, time
+import ctypes.wintypes
+
+from ctypes.wintypes import BOOL
+from ctypes.wintypes import DWORD
+from ctypes.wintypes import HANDLE
+from ctypes.wintypes import LPVOID
+from ctypes.wintypes import LPCVOID
+import win32process
+
+LPCSTR = LPCTSTR = ctypes.c_char_p
+LPDWORD = PDWORD = ctypes.POINTER(DWORD)
+
+
+class _SECURITY_ATTRIBUTES(ctypes.Structure):
+    _fields_ = [('nLength', DWORD),
+                ('lpSecurityDescriptor', LPVOID),
+                ('bInheritHandle', BOOL), ]
```

Comment:
Can probably migrate this to common.py as well or some other shared python file to deduplicate across RTAs